### PR TITLE
API provides visitor ability to only view a part of premium articles

### PR DESCRIPTION
**Pull request by kaylawoodbury**
# [PT Story](https://www.pivotaltracker.com/story/show/171953213)
## User Story
```
As a Visitor 
In order to get a peek of the premium content
I would like to only see the headline and an article snippet for premium articles. 
```
## Changes proposed in this pull request:
* Adds premium_user attribute to user model
* Updates model spec and request spec for premium_user attribute update
* Creates serilaizer spec tests
## What I have learned working on this feature:
* Refresher on how to use rails g for controllers and serializers and migrations to add a new column to the database
* For boolean values, when you validate presence of, the backend will filter our false results and only pass through the true values

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,7 @@
-# frozen_string_literal: true
-
 class User < ActiveRecord::Base
 extend Devise::Models
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
```
Comment (blake-futchi) [inline app/models/user.rb]: format spacing to 2 spaces per indent

File: db/migrate/20200328163747_add_premium_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPremiumToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :premium_user, :boolean
```
Comment (blake-futchi) [inline db/migrate/20200328163747_add_premium_to_user.rb]: do we need to set default boolean to false as well for state management?

**Review by blake-futchi — COMMENTED** [review]
quick formatting change needed and boolean question on migration.

File: app/models/user.rb (hunk 1 above)
Comment (kaylawoodbury) [reply app/models/user.rb]: updating spacing

File: db/migrate/20200328163747_add_premium_to_user.rb (hunk 2 above)
Comment (kaylawoodbury) [reply db/migrate/20200328163747_add_premium_to_user.rb]: added default false for the premium user

**Comment by blake-futchi** [comment]
Good work!

**Pull request merged**